operator/crds/cascading.securecodebox.io_cascadingrules.yaml

```
@@ -1506,6 +1523,7 @@ spec:
               type: object
           required:
           - matches
+          - scanAnnotations
           - scanSpec
```

```suggestion
          required:
          - matches
          - scanSpec
```
The scanAnnotations shouldn't be required otherwise this would result in a breaking change.